src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +55,49 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                INJECTOR.getInstance(MovieSessionService.class);
+
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                INJECTOR.getInstance(AuthenticationService.class);
+        try {
+            System.out.println(authenticationService
+                    .register("Mate@gamil.com", "12345qwert"));
+            System.out.println(authenticationService
+                    .register("Academy@gamil.com", "qwert"));
+            System.out.println(authenticationService
+                    .register("LOLIPOP123f@gamil.com", "098765dfde"));
+        } catch (RegistrationException e) {
+            throw new RuntimeException(e);
```

Comment:
bad practice to throw RuntimeException

Reply:
it's just code for testing, so I didn't want to waste time